src/modules/formBuilder/components/FormBuilder.tsx

```diff
@@ -124,15 +126,24 @@ const FormBuilder: React.FC<FormBuilderProps> = ({
         component='form'
         onSubmit={rhfMethods.handleSubmit(updateForm)}
       >
-        <Box sx={{ flexGrow: 1 }}>
+        <Box
+          sx={{
+            flexGrow: 1,
+            // Can't exceed 100% width of the container minus width of the Palette
+            maxWidth: 'calc(100% - 52px)',
+            display: 'flex',
+            flexDirection: 'column',
+          }}
+        >
           <Box
             sx={{
               // Matches the styles usually applied in DashboardContentContainer.
               // (Moved in here because of the Palette drawer)
-              maxWidth: `${theme.breakpoints.values.lg}px`,
+              maxWidth: `${maxPageWidth}px`,
               pt: 2,
               pb: 8,
               px: { xs: 1, sm: 3, lg: 4 },
+              flexGrow: 1,
```

Comment:
This is to prevent the SaveSlide from creeping up the page, it should stick to the bottom